__docs__/wonder-blocks-theming/tokens-font.mdx

```diff
@@ -0,0 +1,154 @@
+import {Meta} from "@storybook/blocks";
+
+import Banner from "@khanacademy/wonder-blocks-banner";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+
+import TokenTable from "../../.storybook/components/token-table";
+
+<Meta title="Theming / Tokens / Typography" />
+
+export const sampleTextUpper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+export const sampleTextLower = "abcdefghijklmnopqrstuvwxyz";
+
+export const baseColumns = (kind) => [
+    {
+        label: "Token",
+        cell: (row) => (
+            <code>
+                font.{kind}.{row.label}
+            </code>
+        ),
+    },
+    {
+        label: "Value",
+        cell: "value",
+    },
+];
+
+export const mapFunction = ([key, value]) => ({
+    label: key,
+    value: value,
+});
+
+export const tokensFamilyMap = Object.entries(tokens.font.family).map(
+    mapFunction,
+);
+export const tokensWeightMap = Object.entries(tokens.font.weight).map(
+    mapFunction,
+);
+export const tokensSizeMap = Object.entries(tokens.font.size).map(mapFunction);
+export const tokensLineHeightMap = Object.entries(tokens.font.lineHeight).map(
+    mapFunction,
+);
+
+# Tokens
+
+## Typography
+
+Text sizes, weights and font families used across Wonder Blocks.
+
+<Banner
+    kind="warning"
+    text="Note that this is intended only internal
+    development purposes. We'll let you know as soon as this is ready for public
+    consumption."
+/>
+
+### Family
+
+The available font families in our system.
+
+<TokenTable
+    columns={[
+        ...baseColumns("family"),
+        {
+            label: "Example",
+            cell: (row) => (
+                <View
+                    style={{
+                        fontFamily: row.value,
+                    }}
+                >
+                    <span>{sampleTextUpper}</span>
+                    <span>{sampleTextLower}</span>
+                </View>
+            ),
+        },
+    ]}
+    data={tokensFamilyMap}
+/>
+
+### Size
+
+The list of possible font sizes that can be used with our components.
```

Comment:
this might be an issue on my end but i'm not seeing different font sizes? something seems off with the value of `fontSize`.
<img width="884" alt="image" src="https://github.com/Khan/wonder-blocks/assets/77523470/e02fce75-2378-4481-a8a6-81161dc6e69f">

Reply:
ohh good catch! Storybook is messing with the styles, I'll fix that.